### PR TITLE
Show the man pages in the docs

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

File: doc/docs/man/meson.build
Comment (mwperina) [inline doc/docs/man/meson.build:1]: What is this needed for? Isn't the meson build file used only when we translate the man pages from markdown to man format? Don't we here include the markdown files into the HTML documentation directly?

**Review by mwperina — COMMENTED** [review]
Can't the existing markdown files just be linked? If not wouldn't it be nicer to generate HTML files directly (similar what we do with man format) from markdown sources during the build?

**Review by engelmi — COMMENTED** [review]
~~Looks good. Could you also~~
~~- update the path https://github.com/containers/hirte/blob/main/.markdownlint-cli2.yaml#L10 (this fixes also the linter errors)~~
~~- update the path https://github.com/containers/hirte/blob/main/meson.build#L67~~
~~- remove the directory https://github.com/containers/hirte/tree/main/doc/man~~

File: doc/docs/man/meson.build
Comment (engelmi) [reply doc/docs/man/meson.build:1]: > Can't the existing markdown files just be linked? 

Good point. In that case we don't need to update all the other files that I mention [here](https://github.com/containers/hirte/pull/315#pullrequestreview-1443512540). Would be similar to the `api-examples`. 

> What is this needed for? Isn't the meson build file used only when we translate the man pages from markdown to man format? Don't we here include the markdown files into the HTML documentation directly?

Yes, we generate the actual man pages from the markdown files - so the "source of truth" are those md files. I think its good to have them integrated in the readthedocs (which is simple due to the md format).

Comment (pypingou) [reply doc/docs/man/meson.build:1]: that's an oversight, I'll remove the file

**Comment by pypingou** [comment]
> Can't the existing markdown files just be linked?

You mean link with symlink or so? We could but with the approach taken here, we exclude the first two lines from the man pages (ie: the `%hirte 1` that shows in https://github.com/containers/hirte/blob/main/doc/man/hirte.1.md). Something we would not be able to do with simple symlink

**Comment by engelmi** [comment]
> > Can't the existing markdown files just be linked?
> 
> You mean link with symlink or so? We could but with the approach taken here, we exclude the first two lines from the man pages (ie: the `%hirte 1` that shows in https://github.com/containers/hirte/blob/main/doc/man/hirte.1.md). Something we would not be able to do with simple symlink

Wait... my mistake. I thought the actual man pages were copied :sweat: Please ignore my previous comments.

**Review by engelmi — APPROVED** [review]
LGTM

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**